src/main/java/io/cdap/plugin/gcp/gcs/sink/GCSBatchSink.java

```diff
@@ -340,6 +387,73 @@ public void validate(FailureCollector collector) {
       }
     }
 
+    public void validateContentType(FailureCollector failureCollector) {
```

Comment:
please include valid types that are supported per format type. Also it would be good to add this information in plugin doc.

Reply:
Format Types supported are in validation method already

Reply:
Info is in plugin doc already

Reply:
I was referring to javadoc for this public method.